common/txmgr/types/tx_attempt_builder.go

```diff
@@ -15,8 +16,8 @@ import (
 type TxAttemptBuilder[
 	HEAD Head, // HEAD - chain head type
 	FEE Fee, // FEE - chain fee type
-	ADDR any, // ADDR - chain address type
-	TXHASH any, // TXHAHS - chain hash type
+	ADDR types.Hashable[ADDR], // ADDR - chain address type
```

Comment:
I am surprised that this works, but glad that it does :heavy_plus_sign:

Reply:
Yes, me and Aaron were trying, and he came up with this solution. I had no idea such a thing could work.